cmd/todd-agent/main.go

```diff
@@ -51,6 +51,11 @@ func main() {
 
 	cfg := config.GetConfig(arg_config)
 
+	// Create optDir
+	if err := os.MkdirAll(cfg.LocalResources.OptDir, 0755); err != nil {
```

Comment:
Makes sense. fwiw, I use [goxc](https://github.com/laher/goxc) to generate the [releases](https://github.com/vcabbage/trivialt/releases) for trivialt. Handy for the simple binary case since it builds and uploads archives to github.